### DISCUSSION TITLE
Meeting 2024-11-19

### DISCUSSION THREAD

**Discussion by adamgreig** [rust-embedded/wg#805] [General]
This is the agenda for the next meeting. Please add anything you'd like to discuss below!

* Meetings: Tuesday 8pm Europe/Berlin time
* [Join the Matrix chat](https://matrix.to/#/#rust-embedded:matrix.org)
* [Nominated issues](https://github.com/search?q=org%3Arust-embedded+label%3Anominated+is%3Aopen&type=Issues)
* [IRC/Matrix logs](https://libera.irclog.whitequark.org/rust-embedded/2024-11-19)

**Comment by MabezDev** [comment]
I'd like to continue the discussion of support of 0.2 eha, and what to do with https://github.com/rust-embedded/embedded-hal/pull/640

**Comment by BartMassey** [comment]
Philipp Oppermann and I have been working with/through https://github.com/rust-embedded/rust-raspberrypi-OS-tutorials/ with the goal of eventually doing a blog along the lines of his previous blog [Writing an OS In Rust](https://os.phil-opp.com/) but for the Raspberry Pi Zero 2 W.

I'd like to discuss the state of this material, what we've done and found so far, and try to figure out the best way for us to proceed.

**Comment by hdoordt** [comment]
Announcement: next book sprint for the Discovery MB2 book will be on January 25, 2025, 17:00-21:00 CEST, 15:00-19:00 UTC, 8:00AM-12:00PM PST. Join if you're interested in helping to write the new discovery book chapter on DMA and PWM!

If you're interested in joining, you can make that clear by liking this message.
